content/how-to/cloud-servers/multifactor-authentication-from-the-cloud-control-panel/index.md

```diff
@@ -191,28 +211,23 @@ account.
 
 1. Log in to the [Rackspace Cloud Control Panel](https://login.rackspace.com).
 
-2. In the top navigation bar, click **Select a Product > Rackspace Cloud**.
+2. In the upper-right, click **Account** > **Account Settings**.
 
-3. In the upper-right, click **Account** > **Account Settings**. Under
-   **Rackspace Account Settings**, click **Change** next to **Multi-Factor
-   Authentication**.
+   <img width="311" alt="3 2" src="https://user-images.githubusercontent.com/96761530/161067080-ed2759d3-b497-4b4d-a632-2726dbf023c9.png">
```

Comment:
Perhaps we can change the account number to not be a real number, cover it up somehow or blur somehow.
![image](https://user-images.githubusercontent.com/14296817/166593380-44567d2e-1457-4e7c-9fd3-1f6468ec5310.png)

Reply:
Updated....